incubator/hnc/third_party/open-policy-agent/gatekeeper/pkg/webhook/certs.go

```diff
@@ -84,8 +90,13 @@ type CertRotator struct {
 	CAName          string
 	CaOrganization  string
 	DNSName         string
-	CertsMounted    chan struct{}
+	IsReady         chan struct{}
+	certsMounted    chan struct{}
 	certsNotMounted chan struct{}
+	caInjected      bool
+	caNotInjected   chan struct{}
```

Comment:
Nit: These have parallel names so it's a bit weird for them to have different types. Maybe use `wasCAInjected` to highlight that it's a boolean?

Reply:
done.